src/createSaleorTheme/types.ts

```diff
@@ -0,0 +1,79 @@
+/* eslint-disable @typescript-eslint/unified-signatures */
```

Comment:
What is this for?

Reply:
https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/unified-signatures.md

It's from mui types to enable function polymorphism